scripts/manpage_generation.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+
+
+# gather the path information
+manpage_script_path = os.path.abspath(__file__)
+
+convert2rhel_path = os.path.dirname(os.path.dirname(manpage_script_path))
+
+sys.path.append(convert2rhel_path)
+
+# Directory to store the generated manpages
+MANPAGE_DIR = "man"
+
+print("Generating manpages")
+
+# Detect changes in __init__.py
+result = subprocess.run(["git", "diff", "--quiet", "HEAD~1", "--", "convert2rhel/__init__.py"], check=True)
+
+if result.returncode == 0:
+    # No changes detected, exit with 0
+    print("No new version detected")
+    sys.exit(0)
+else:
+    print("Changes detected")
+
+    subprocess.run(["dnf", "install", "-y", "python3-rpm"], check=True)
+    subprocess.run(["dnf", "install", "-y", "python3-dnf"], check=True)
```

Comment:
```suggestion
    subprocess.run(["dnf", "install", "-y", "python3-rpm", "python3-dnf"], check=True)
```

Reply:
Thank you for the extra look on this, I will get this taken care of and push it up!